plugins/com.google.cloud.tools.eclipse.appengine.deploy/src/com/google/cloud/tools/eclipse/appengine/deploy/standard/StandardDeployJob.java

```diff
@@ -112,12 +113,16 @@ public IStatus runInWorkspace(IProgressMonitor monitor) throws CoreException {
           return cloudSdkProcessStatus;
         }
         // temporary way of error handling, after #439 is fixed, it'll be cleaner
-        return StatusUtil.error(getClass(), "Staging failed, check the error message in the Console View");
+        String errorMessage =
+            getErrorMessageOrDefault(Messages.getString("deploy.job.staging.failed"));
+        return StatusUtil.error(getClass(), errorMessage);
```

Comment:
Unfortunately, this doesn't seem to show the error message when staging fails:

![selection_007](https://cloud.githubusercontent.com/assets/10523105/21897876/e0051a90-d8b8-11e6-854c-cf53edeec9cc.png)

FYI, to reproduce staging failure, change `appengine-web.xml` to be a malformed XML.